src/main/java/com/radixdlt/identifiers/Magical.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.identifiers;
+
+public interface Magical {
```

Comment:
This seems a bit heavy handed to me.  For now I think the simplest thing that would work would be to pass in the magic number to constructors or factory methods.  Not really convinced that we need an interface for something that returns a number.

Reply:
Yeah sure! Removed :)